test/table.js

```diff
@@ -1759,6 +1772,18 @@ describe('Bigtable/Table', function() {
         });
       });
     });
+
+    it('should use an appProfileId', function(done) {
```

Comment:
This test looks perfect 👍 Could you just bump it up a bit, so that it is directly after the test called `'should provide the proper request options'`?